modules/apps/headless/headless-builder/headless-builder-impl/src/main/resources/com/liferay/headless/builder/internal/batch/headless-builder.json

```diff
@@ -484,16 +484,16 @@
 				{
 					"deletionType": "prevent",
 					"label": {
-						"en_US": "API Schemas API Properties"
+						"en_US": "API Schema API Properties"
 					},
-					"name": "apiSchemasAPIProperties",
+					"name": "apiSchemaAPIProperties",
 					"objectDefinitionExternalReferenceCode1": "MSOD_API_SCHEMA",
 					"objectDefinitionExternalReferenceCode2": "MSOD_API_PROPERTY",
 					"objectDefinitionName2": "APIProperty",
 					"parameterObjectFieldId": 0,
 					"parameterObjectFieldName": "",
 					"reverse": false,
-					"type": "manyToMany"
+					"type": "oneToMany"
```

Comment:
Are we sure about this? According to the research that we made, a property could belong to 1 or many schemas and a Schema could contain one or many properties. That's why it was marked as a many-to-many relationship

This way, we will consider each property belongs to one and only one schema. I am not saying is wrong, I'm just saying that this wasn't our agreement on the research so, if we change this, we should update the research document just to keep updated the documentation.

Reply:
Why would a property be in different schemas? Carlos and I could not find a good reason even looking at the confluence document.

Reply:
I think, even if conceptually it is possible, we should be fetching the existing properties when creating a new schema in case it matches exactly with the existing property